scripts/stack/configure-stack.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+set -o pipefail
+
+if [[ ${CURDIR:-} == "" ]]; then
+    export CURDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+fi
+source $CURDIR/read-env.sh $CURDIR/.env
+
+PYTHON_EXECUTABLE=""
+
+if which python3 > /dev/null; then
+    PY_VERSION=`python3 --version`
+    PYTHON_EXECUTABLE="python3"
+elif which python > /dev/null; then
+    PY_VERSION=`python --version`
+    PYTHON_EXECUTABLE="python"
+fi
```

Comment:
We actually don't run from system python, rather from venv version. Do you think it can cause trouble?

Reply:
This is only used if a `make clean install` is needed (if the user never built the `bin/connectors` from the source) - which, the `make` will create a venv when it does its magic.